packages/devops/scripts/ci/pushBuilds.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -x
+
+PACKAGE=$1
+DIR=$(dirname "$0")
+${DIR}/copyFromCommonVolume.sh # pull in .env files and builds from common volume
+if [ -f "/common/deployment_exists" ]; then
+    echo "Deployment for ${CI_BRANCH} exists, updating with latest builds"
+    DEPLOYMENT_SSH_URL=$(${DIR}/determineDeploymentSshUrl.sh)
+    cd /
+    tar -zcf tupaia.tar.gz /tupaia
+    scp -o StrictHostKeyChecking=no -p /tupaia.tar.gz ubuntu@$DEPLOYMENT_SSH_URL:./tupaia.tar.gz
+    ssh -o ServerAliveInterval=15 ubuntu@$DEPLOYMENT_SSH_URL "tar -zxf tupaia.tar.gz && rm tupaia.tar.gz"
```

Comment:
Zip, push via scp, then unzip on the server. Tried a few ways and this (although slow at around 8 minutes) was the fastest. I have some thoughts around ways we can improve this in the future, but this keeps it at around the same total build time (~30 mins when the docker image is fully cached) as we have currently, but with the builds happening off the machine so they don't consume resources and should be more stable